packages/fetchai/skills/erc1155_client/tests/test_behaviours.py

```diff
@@ -34,7 +34,7 @@ class TestSearchBehaviour(ERC1155ClientTestCase):
     def test_setup(self):
         """Test the setup method of the search behaviour."""
         # operation
-        self.search_behaviour.setup_class()
+        self.search_behaviour.setup()
```

Comment:
Why the change? `search_behaviour` is not a test class...

Reply:
because pycharm search and replace messed this up, it never had a setup_class method to begin with